plone/app/contenttypes/browser/templates/folder_listing.pt

```diff
@@ -41,7 +41,7 @@
                         isAnon portal_state/anonymous;
                         normalizeString nocall: context/plone_utils/normalizeString;
                         toLocalizedTime nocall: plone_view/toLocalizedTime;
-                        show_about python:not isAnon or site_properties.allowAnonymousViewAbout;
+                        show_about python:not isAnon or context.portal_registry['plone.allow_anon_views_about'];
```

Comment:
+1 for a method in plone.app.layout since you can use that in skin-templates as well.